### ISSUE TITLE
Re-render child nodes if they are modified in a different Node's render task.

### ISSUE THREAD

**Issue by trusktr** [infamous/infamous#17]
f.e.,

```js
let r = 0
node1.addRenderTask(function() {
  r++
  node1.rotation = [0,r,0]
  node2.rotation = [0,r,0]
})
```

In the example and with the current API, `node2` isn't rendered because this task is registered from node1 and we currently only render the node associated with the task.

### Idea 1:

We might be able to find the top-most common ancestor of all nodes that were updated, then traverse the tree to render all children, in the process of which unsetting children from needing to be rendered.

### Idea 2:

Or we could keep it as is, and make it clear that a separate render task has to be specified for each node that we wish to have updated. This would make things more verbose (we'd need to use the `addRenderTask` method a lot more), but that might possibly make it more explicit what updates. There may be performance implications when having a bigger list of render tasks to fire. The previous example would need to be written as follows (this works right now):

```js
let r = 0
node1.addRenderTask(function() {
  r++
  node1.rotation = [0,r,0]
})
node2.addRenderTask(function() {
  node2.rotation = [0,r,0]
})
```

Related: #13

**Comment by trusktr** [comment]
This will be solved by #23.

**Comment by trusktr** [comment]
Solved in ffe311a2bc9c8741428d6fe60ae59bb10b7160f1

Neither of above ideas were used. Instead, we add render tasks to the Motor, and it keeps track of which Nodes are modified whenever a Node's setter is used inside (or even outside) of the render task.
